docs/getting_started/installation.rst

```diff
@@ -6,10 +6,50 @@ Installation
 Before you start
 ================
 
-A basic Wagtail setup can be installed on your machine with only a few prerequisites - see `A basic Wagtail installation`_. However, there are various optional components that will improve the performance and feature set of Wagtail, and our recommended software stack includes the PostgreSQL database, ElasticSearch (for free-text searching), the OpenCV library (for image feature detection), and Redis (as a cache and message queue backend). This would be a lot to install in one go, and for this reason, we provide a virtual machine image, for use with `Vagrant <http://www.vagrantup.com/>`__, with all of these components ready installed.
+You can get basic Wagtail setup installed on your machine with only a few prerequisites. See the full  `Dependencies`_ list below.
 
-Whether you just want to try out the demo site, or you're ready to dive in and create a Wagtail site with all bells and whistles enabled, we strongly recommend the Vagrant approach. Nevertheless, if you're the sort of person who balks at the idea of downloading a whole operating system just to run a web app, we've got you covered too. Start from `A basic Wagtail installation`_ below.
+There are various optional components that will improve the performance and feature set of Wagtail. Our recommended software stack includes the PostgreSQL database, ElasticSearch (for free-text searching), the OpenCV library (for image feature detection), and Redis (as a cache and message queue backend). This would be a lot to install in one go. For this reason we provide a virtual machine image to use with `Vagrant <http://www.vagrantup.com/>`__, with all of these components ready installed.
 
+Whether you just want to try out the demo site, or you're ready to dive in and create a Wagtail site with all bells and whistles enabled, we strongly recommend the Vagrant approach. Nevertheless, if you're the sort of person who balks at the idea of downloading a whole operating system just to run a web app, we've got you covered too. Start from `Install Python`_.
+
+
+Dependencies
+============
+
+The basic Wagtail installation is pure Python. No build tools are required on the host machine.
+If you are installing Wagtail differently (e.g. from the Git repo), make you meet the following dependencies:
+
+Required
+--------
+
+ * Django>=1.7.0,<1.8
```

Comment:
Thanks, @kaedroho. Changed with a link to setup.py for the full list of dependencies + added the note on running `python setup.py install``